docs/modules/automation-testing/pages/scripting/create-image-injection-script.adoc

```diff
@@ -0,0 +1,98 @@
+= Create image injection script
+:navtitle: Create an image injection script
+
+Learn how to automate the process of image injection, so you can automate your test suite.
+
+== Before you start
+
+Enable image injection for your app with either of these options:
+
+* Either use the capability `'kobiton:instrumentenabled'` to xref:#appium_instrumentation_capability[instrument] your app.
+
+* Or add Kobiton image injection SDK into your Android or iOS apps. Follow xref:apps:image-injection-sdk/add-the-sdk-to-your-android-app.adoc[this guide] for andriod, and xref:apps:image-injection-sdk/add-the-sdk-to-your-ios-app.adoc[this guide] for iOS.
+
+[#appium_instrumentation_capability]
+== Appium capability (instrumentation only)
+
+Add the `'kobiton:instrumentenabled'` capability to your script’s capabilities list. The value of this capability must be `true` to instrument the app for image injection.
+
+
+[IMPORTANT]
+Only include this Appium capability for image injection via app instrumentation. *Do not* include it for apps with image injection SDK embedded.
+
+== Kobiton commands

```

Comment:
@tungmhoang Is this section for both Instrumenting and SDK or SDK only? the first sentence mentions SDK. Are there different commands for instrumenting? Or should the first sentence be updated?

Reply:
remove the reference to SDK and mention the commands work for both instrumentation and SDK